flow/flow.go

```diff
@@ -358,9 +358,10 @@ func SystemInit(args *Config) error {
 	var cpus []uint
 	var err error
 	if args.CPUList != "" {
-		if cpus, err = common.ParseCPUs(args.CPUList, CPUCoresNumber); err != nil {
+		if cpus, err = common.ParseCPUs(args.CPUList); err != nil {
 			return err
 		}
+		cpus = common.DropInvalidCPUs(cpus, CPUCoresNumber)
```

Comment:
Please ask Ilya or Gregory about CPUCoresNumber as max cpu number?

Reply:
NumCPU - OK, however I suppose that all user input should be handled in one function. Parsing, duplication, trunking, any other points.